### PR TITLE
Update change log for rc.2

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary
Update changelog for RC.2

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [NA] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [NA] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [NA] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [NA] [Add `[feature]` if the change is significant or affectes feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,80 @@
...
+- Make 'AllSigned' execution policy require modules under $PSHome to be signed (#5511)
+
+### Engine updates and fixes
+
+- Update powershell to use `2.0.4` dotnet core runtime. (#5677)
+- Remove references to the old executable `powershell` or `powershell.exe` (#5408)
+
+### General cmdlet updates and fixes
+
+- Remove unncessary check for Paths.count > 0, in the `*-FileCatalog` CmdLets, as there is code later to use the current (#5596)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: The clause after the comma seems to end abruptly.  I don't think it's even needed at all.  Can just put a period after `Cmdlets`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,80 @@
...
+- Use explicit libpsl-native binary name for dllimport. (#5580)
+
+### Build and Packaging Improvements
+
+- Fix Get-EnvironmentInformation to properly check for CoreCLR (#5592) (Thanks @markekraus!)
+- Make Travis CI use `libcurl+openssl+gssapi` (#5629) (Thanks @markekraus!)
+- Disambiguate icon for daily builds on Windows (#5467) (Thanks @bergmeister!)
+- Fix Import-CliXml tests which still use `powershell` instead of `pwsh` and make sure it fails if it regresses (#5521) (Thanks @markekraus!)
+- Update port used to WebCmdLet test which broke due to a change in AppVeyor (#5520) (Thanks @markekraus!)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: `Update port number used for WebCmdlet tests ..."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,80 @@
...
+- Clean up use of 'Runspaceconfiguration' from comments and xunit test code (#5569) (Thanks @Bhaal22)
 - Remove Pester as a module include with the PowerShell Packages.
   In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Remove `In the future`, they should be able to do this today for their own tests.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,80 @@
...
-- Make Travis CI use `libcurl+openssl+gssapi` (#5629, @markekraus)
 - Refactor `New-UnixPackaging` into functions to make the large function more readable. (#5625)
+- Make the experience better when start-pspester doesn't find pester (#5673)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Pascal casing for cmdlet

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,12 +1,80 @@
...
+- Add linux-arm tarball package to release build (#5652, #5660)
+- Enable building for win-arm and win-arm64 (#5524)
+- Make macOS package require 10.12 or newer (#5649, #5654)
+- Update signing subjects to something meaningful (#5650)
+- Make `New-UnixPackage` more readable (#5625)
+- Update `PowerShellGet` tests to validate the new install location of AllUsers scope. (#5633)
+- Increase reliability of flaky test that fails intermittently in CI (#5641)
+- Exclude markdown files from 'Pester' folder from the Markdown meta test (#5636)
+- Run tests for Windows installer only on Windows (#5619)
+- Supress the expected errors from Select-Xml tests (#5591)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Suppress

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,12 +1,80 @@
...
+- Make the -SslProtocol tests pending (#5605)
+- Add retry logic to prerequisite URL and output URL on failure so you can more easily troubleshoot (#5601, #5570)
+- Replace HttpListener Response Tests with WebListener (#5540)
+- Make sure submodule are initialized when running Mac release build (#5496)
+- Update version regex for nanoserver docker
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: missing PR #?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,12 +1,80 @@
...
+- Enable building for win-arm and win-arm64 (#5524)
+- Make macOS package require 10.12 or newer (#5649, #5654)
+- Update signing subjects to something meaningful (#5650)
+- Make `New-UnixPackage` more readable (#5625)
+- Update `PowerShellGet` tests to validate the new install location of AllUsers scope. (#5633)
+- Increase reliability of flaky test that fails intermittently in CI (#5641)
+- Exclude markdown files from 'Pester' folder from the Markdown meta test (#5636)
+- Run tests for Windows installer only on Windows (#5619)
+- Supress the expected errors from Select-Xml tests (#5591)
+- Make the -SslProtocol tests pending (#5605)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: This one is not needed.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,12 +1,80 @@
...
+- Add retry logic to prerequisite URL and output URL on failure so you can more easily troubleshoot (#5601, #5570)
+- Replace HttpListener Response Tests with WebListener (#5540)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: Was this done by `markekraus`? If so, we should mention him.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,12 +1,80 @@
...
+- Make sure submodule are initialized when running Mac release build (#5496)
+- Update version regex for nanoserver docker
+- Remove duplicate files in Windows packages in a folder called `signed` (#5527)
+- Add PowerShell VSCode style settings (#5529) (Thanks @bergmeister)
+- Add Travis CI matrix for improved job tagging (#5547)
+- Fix the path to 'powershell_xxx.inc' in Start-Build (#5538)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: This is one by Ilya and we should mention him.

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
@TravisEz13 You probably need to put many words in code blocks (``) to avoid the spelling checks for them, for example, cmdlet names, file names, arm32 and arm64 and etc.

File: CHANGELOG.md (hunk 1 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 2 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 3 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 4 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 5 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved with other spelling issues

File: CHANGELOG.md (hunk 6 above)
Comment (TravisEz13) [reply CHANGELOG.md]: Not a PR, removed

File: CHANGELOG.md (hunk 8 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md (hunk 9 above)
Comment (TravisEz13) [reply CHANGELOG.md]: resolved

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,12 +1,79 @@
...
 
-- Update the contribution guideline to note that updating the changelog is required. (#5586)
+### Breaking changes
+
+- Skip null-element check for collections with a value-type element type (#5432)
+- Make `AllSigned` execution policy require modules under `$PSHome` to be signed (#5511)
+
+### Engine updates and fixes
+
+- Update powershell to use `2.0.4` dotnet core runtime. (#5677)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: When `powershell` is used to mean the product, we should capitalize: PowerShell

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,12 +1,79 @@
...
+- Use explicit `libpsl-native` binary name for `dllimport`. (#5580)
+
+### Build and Packaging Improvements
+
+- Fix `Get-EnvironmentInformation` to properly check for CoreCLR (#5592) (Thanks @markekraus!)
+- Make Travis CI use `libcurl+openssl+gssapi` (#5629) (Thanks @markekraus!)
+- Disambiguate icon for daily builds on Windows (#5467) (Thanks @bergmeister!)
+- Fix `Import-CliXml` tests which still use `powershell` instead of `pwsh` and make sure it fails if it regresses (#5521) (Thanks @markekraus!)
+- Update port number used for WebCmdlets tests which broke due to a change in AppVeyor (#5520) (Thanks @markekraus!)
+- Clean up use of `Runspaceconfiguration` from comments and xUnit test code (#5569) (Thanks @Bhaal22)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: No exclamation after Bhaal22?

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,12 +1,79 @@
...
+- Replace HttpListener Response Tests with WebListener (#5540) (Thanks @markekraus!)
+- Fix the path to `powershell_xxx.inc` in Start-Build (#5538) (Thanks @iSazonov!)
 - Remove Pester as a module include with the PowerShell Packages.
-  In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
-- Make Travis CI use `libcurl+openssl+gssapi` (#5629, @markekraus)
+  You should be able to add it by running `Install-Module Pester`. (#5623, #5631)
 - Refactor `New-UnixPackaging` into functions to make the large function more readable. (#5625)
+- Make the experience better when `Start-PSPester` doesn't find Pester (#5673)
+- Update packaging and release build scripts to produce zip packages for `win-arm` and `win-arm64`∏ (#5664)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: looks like an extra symbol after `win-arm64`

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**